Tests/test_raxml_tool.py

```diff
@@ -24,7 +25,8 @@
 
 if not raxml_exe:
     raise MissingExternalDependencyError(
-        "Install RAxML (binary raxmlHPC) if you want to test the Bio.Phylo.Applications wrapper.")
+        "Install RAxML (binary raxmlHPC) if you want to test the Bio.Phylo.Applications wrapper."
```

Comment:
@peterjc  should this line be split in two?, it exceeds 88; otherwise is fine to be merged.
```
        "Install RAxML (binary raxmlHPC) if you want to test the Bio.Phylo.Applications"
        " wrapper."
```

Reply:
Black doesn't (yet) deal with breaking/joining strings, but yes it would be nice to fix that by hand here.